core/drivers/crypto/crypto_api/acipher/local.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2020 NXP
+ *
+ * Brief   Definition of the functions shared locally.
+ */
+#ifndef __LOCAL_H__
+#define __LOCAL_H__
+
+#include <drvcrypt_acipher.h>
+
+/*
+ * Mask Generation function. Use a Hash operation
+ * to generate an output @mask from a input @seed
+ *
+ * @mgf_data  [in/out] MGF data
+ */
+TEE_Result rsa_mgf1(struct drvcrypt_rsa_mgf *mgf_data);
```

Comment:
Even if these symbols are local to acipher/ ?

Reply:
@clementfaure unless they're `static` functions or variables, the symbols will share the same global namespace in the end.
So if you can make this static, it's best, otherwise a prefix if the way we keep things tidy and avoid collisions.

Reply:
Ok, no problem